### ISSUE TITLE
problem with configure prefix 

### ISSUE THREAD

**Issue by remyd1** [apptainer/singularity#26]
Hi,

I found another issue. When I try to install singularity in another directory than the default one (/opt/singularity), I have issues with the command "/opt/singularity/bin/singularity build" once it is installed :

```
/opt/singularity/libexec/singularity/mods/install.smod: line 47: /usr/libexec/singularity/ftype: No such file or directory
... a lot of that kind of message ...
```

A little fix is to add a symbolic link from where singularity is searching :

``` bash
mkdir -p /usr/libexec && cd $_
ln -s /opt/singularity/libexec/singularity
```

I think that this problem appears since the last update in the master branch.

Best,
Remy

**Comment by gmkurtzer** [comment]
Ooops, that was a copy paste mess up! I should have given the past h to ftype referencing $libexecddir! I'll fix as soon as I'm at a real keyboard. Thanks!

**Comment by gmkurtzer** [comment]
This should be fixed in master e7409ff. Thanks again!
